src/typescript-service.ts

```diff
@@ -51,37 +52,37 @@ export class TypeScriptService implements LanguageHandler {
 		this.traceModuleResolution = traceModuleResolution || false;
 	}
 
+	// TODO pass remoteFs and strict to constructor
```

Comment:
Nope, I misread this as meaning "pass remoteFs and strict to the ProjectManager constructor" and saw that they were already passed and the TODO was satisfied.